app/views/comments/_comment_upvote_form.html.erb

```diff
@@ -0,0 +1,10 @@
+<%= form_with(url: comment_liking_relations_path,
+              scope: 'comment_liking_relation',
+              method: data[:method], class: 'd-inline-block',
+              data: { turbo_frame: "comment-#{index}-points" }) do |f| %>
+    <%= f.hidden_field :liked_comment_id, value: comment.id %>
+    <%= f.hidden_field :liking_user_id, value: current_user&.id %>
```

Comment:
tu chyba podobny case co kiedyś? Czy liking_user_id nie powinno pobierać w kontrolerze z sesji dla bezpieczenstwa?

Reply:
nie wiem do konca jak dzialaja te hidden fields w rails, ale czy w network tab w przegladarce ten liking_user_id tez jest wysylany? i ktoś złośliwy by podmienił?

Reply:
Tak, masz rację, naprawiłem to chyba wcześniej w kontrolerze, ale zapomniałem usunąć hidden_field.